app/src/main/kotlin/com/wire/android/util/ui/WireSessionImageLoader.kt

```diff
@@ -16,6 +17,7 @@ import com.wire.kalium.logic.feature.asset.GetMessageAssetUseCase
  * As it uses Kalium's [GetAvatarAssetUseCase], a loader created for one session may be unable to load images from another session.
  * It wraps Coil, so it becomes easier to refactor in the future if we ever switch from Coil to something else.
  */
+@Stable
```

Comment:
while looking into the Issue I noticed that the UserAvatar gets recomposoed each time the item is visible due to Compose being unable to check for the stability of this class so I added @Stable annotation that promises the Composer that you do not need to check if it is stable, because it always is. It should add some performance gain I think and I do not think it can hurt us or cause any issues